Client/Frontend/ContentBlocker/ContentBlockerSettingViewController.swift

```diff
@@ -6,7 +6,38 @@ import Foundation
 import Shared
 
 @available(iOS 11.0, *)
-class ContentBlockerSettingViewController: SettingsTableViewController {
+class ContentBlockerSettingsTableView : SettingsTableViewController {
+    // The first section header gets a More Info link
+    override func tableView(_ tableView: UITableView, viewForFooterInSection section: Int) -> UIView? {
+        if section == 0 {
+            return super.tableView(tableView, viewForFooterInSection: section)
+        }
+
+        // TODO: Get a dedicated string for this.
+        let title = NSLocalizedString("More Info…", tableName: "SendAnonymousUsageData", comment: "Re-using more info label from 'anonymous usage data' item for showing a 'More Info' link on the Tracking Protection settings screen.")
+
+        var attributes = [String: AnyObject]()
+        attributes[NSFontAttributeName] = UIFont.systemFont(ofSize: 12, weight: UIFontWeightRegular)
+        attributes[NSForegroundColorAttributeName] = UIConstants.HighlightBlue
+
+        let button = UIButton()
+        button.setAttributedTitle(NSAttributedString(string: title, attributes: attributes), for: .normal)
+        button.contentHorizontalAlignment = .left
+        // Top and left insets are needed to match the table row style.
+        button.contentEdgeInsets = UIEdgeInsetsMake(8, 16, 0, 0)
+        button.addTarget(self, action: #selector(ContentBlockerSettingsTableView.moreInfoTapped), for: .touchUpInside)
+        return button
+    }
+
+    func moreInfoTapped() {
+        let viewController = SettingsContentViewController()
+        viewController.url = SupportUtils.URLForTopic("tracking-protection-ios")
```

Comment:
We need make sure this gets QA verified. Note to self to set flag in bug for that